documentation/docs/migration-guide.md

```diff
@@ -90,4 +93,54 @@ There are a few other noticeable changes from `@cognite/3d-viewer` and `@cognite
 
 - `@cognite/3d-viewer` supports local caching to reduce the time to load previously opened 3D models. Currently, this is not supported by `@cognite/reveal`, but the need for such functionality is reduced by adding streaming capabilities.
 - In `@cognite/3d-viewer` `Cognite3DViewer.addModel(...)` will always return a `Cognite3DModel`. In `@cognite/reveal` this function might also return a `CognitePointCloudModel`. To explicitly add a CAD model or point cloud model use `Cognite3DViewer.addCadModel(...)` or `Cognite3DViewer.addPointCloudModel(...)`
-- `Cognite3DViewer.loadCameraFromModel(...)`] has been added for loading camera settings from CDF when available.
\ No newline at end of file
+- `Cognite3DViewer.loadCameraFromModel(...)`] has been added for loading camera settings from CDF when available.
+
+# Preparing models
```

Comment:
```suggestion
## Preparing models
```

You shouldn't have multiple h1 titles on a single page. It's semantically incorrect.